src/test/shell/bazel/bazel_coverage_test.sh

```diff
@@ -136,21 +156,10 @@ end_of_record"
 }
 
 function test_cc_test_coverage_gcov() {
-  local -r gcov_location=$(which gcov)
-  if [[ ! -x ${gcov_location:-/usr/bin/gcov} ]]; then
-    echo "gcov not installed. Skipping test."
-    return
+  if is_gcov_right_version_installed; then
+    echo "Skipping test." && return
```

Comment:
This reads wrong, right? It should be if wrong_gcov_is_installed then echo "skipping test". Or am I off?

Reply:
Pfff yes, I got so used to the code I totally missed it. Thanks!